tasmota/tasmota_template.h

```diff
@@ -176,6 +176,7 @@ enum UserSelectablePins {
   GPIO_BL0942_RX,                      // BL0942 Serial interface
   GPIO_HM330X_SET,                     // HM330X SET pin (sleep when low)
   GPIO_HEARTBEAT, GPIO_HEARTBEAT_INV,
+  GPIO_SDM230_TX, GPIO_SDM230_RX,      // SDM230 Serial interface
```

Comment:
Needs to be added at end of list

Reply:
Moved to the end. By the way - what's the purpose for that?